### DISCUSSION TITLE
Javascript errors on netscape accessing google.com

### DISCUSSION THREAD

**Discussion by ssshake** [atauenis/webone#37] [General]
Hello there, my issue has two purposes. 

1) is to report this issue and get help. 

2) Is to introduce myself because I couldn't find out how to contact you otherwise.

Regarding the issue when I go to google through your proxy on netscape 4 I get these javascript parsing errors just like if I went to google directly. 

![image](https://user-images.githubusercontent.com/1675868/114599382-05eb4e80-9c61-11eb-9f73-62310e0a0c58.png)

I saw mention of your service transpiling JS for older system. If this is true is there something I have to turn on to do this transpiling? Alternatively is there a way I can strip out the javascript?

As for the introduction. My name is Richard Bettridge and I created theoldnet.com two years ago. I have similar goals in mind as you. I want to be able to access various internet resources on my collection of vintage computers.

Over the past year or so I have been speaking to other people who also have this goal in mind. There people have been creating their own tools and services. 

I would like us all to know each other, and to help each other in some form or another. We are all working on very similar stuff, I think it's perhaps wasteful for us to work on each piece in isolation. 

I am really excited about what you are doing here, some of the features are really cool. I want to know more, I want to use your service and I want to help where I can. 

If you have any interest in speaking to me and the group please let me know how best to contact you!

**Comment by atauenis** [comment]
Hello. Your project is looking interesting. Looks like an time machine. But Kaspersky Anti-Virus is blocking some pages on it, saying that "the site can steal data". May be because it is a proxy. :)

About the JavaScript adaptions. WebOne can edit scripts, but only when have find&replace masks. Just like download a page, edit in in Notepad/GEdit/Kate/etc and then open edited file in your browser. Every site requires custom patches (edit sets), it is not possible to make edit sets which will fix all stuff on all sites. Currently default webone.conf is containing only some common CSS fixes and one example JS fix (for http://old-dos.ru website). I have not tried to write edit set for Google.com as this needs a large time and because Netscape have no "Developer tools" like in new browsers, so I can't quickly find all "errors".

**Reply by ssshake** [reply]
Great thanks for the feedback I'll follow up with kasperky. 

I did not know github had a discussion page I should have created this here. 

Could you tell me one url that Kaspersky complains about? I will use that in my false positive report.

**Reply by atauenis** [reply]
http://theoldnet.com/get?url=w3c.org&year=1996&scripts=false&decode=false

![theoldnet](https://user-images.githubusercontent.com/4800266/114604373-79ad4b80-9ca1-11eb-945a-40da4b91f792.png)


Translated:
> Navigation to site which can be dangerous for your data has prevented.
> Breaked navigation to a phishing or a fake site, created to force you to write passwords, credit card info or other personal data.
> "Kaspersky Laboratory" saved you from going onto this website. You can safely close it.

If they want full error log (in Russian), there is it:
```
Событие: Переход остановлен
Имя программы: chrome.exe
Путь к программе: C:\Program Files (x86)\Google\Chrome\Application
Компонент: Веб-Антивирус
Описание результата: Запрещено
Тип: Угроза потери данных
Название: http://theoldnet.com/get?url=w3c.org&year=1996&scripts=false&decode=false
Точность: Точно
Степень угрозы: Высокая
Тип объекта: Веб-страница
Имя объекта: get?url=w3c.org&year=1996&scripts=false&decode=false
Путь к объекту: http://theoldnet.com
Причина: Облачная защита
```

**Reply by atauenis** [reply]
> If you have any interest in speaking to me and the group please let me know how best to contact you!

About contacts with me. I'm prefer Vogons (topics and private messages) and GitHub Issues/Discussions.

**Reply by ssshake** [reply]
Thanks for the info!

Can I find you under the same name on vogons?

**Reply by atauenis** [reply]
https://www.vogons.org/viewtopic.php?f=24&t=67165
https://www.vogons.org/memberlist.php?mode=viewprofile&u=38157
